lib/Results.js

```diff
@@ -19,7 +19,7 @@ class Players extends Array {
         // run out of ram allocating these objects.
         num = Math.min(num, 10000);
 
-        while(this.players.length < num) {
```

Comment:
Wow, this is... definitely broken. Surprised nobody's stumbled on this earlier. Thanks!